specification/synapse/resource-manager/readme.azureresourceschema.md

```diff
@@ -46,3 +47,23 @@ input-file:
   - Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json
 
 ```
+### Tag: schema-synapse-2020-12-01 and azureresourceschema
+
+``` yaml $(tag) == 'schema-synapse-2020-12-01' && $(azureresourceschema)
+output-folder: $(azureresourceschema-folder)/schemas
+
+# all the input files in this apiVersion
+input-file:
```

Comment:
keys.json is missing here. Is that intentional?

Reply:
Good catch. Added it.